types/koa-websocket/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
+        "target": "es6",
```

Comment:
Why do you need to change this?

Reply:
It was required to iterate over `Set` in the test. But I've noticed now that is not necessary .

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/29611/files#diff-f63da9cb877245f2161aa3bb12520498